README.md

```
@@ -13,19 +14,19 @@
   </picture>
 </p>
 
-This provides three Physical Simulators for *Silicon Dangling Bond (SiDB)* Logic: [*ClusterComplete*](https://www.math.ru.nl/~bosma/Students/WillemLambooyMSc.pdf), [
-*QuickExact*](https://www.cda.cit.tum.de/files/eda/2024_aspdac_efficient_exact_simulation.pdf)
-and [*QuickSim*](https://www.cda.cit.tum.de/files/eda/2023_ieeenano_quicksim_physical_simulation.pdf).
+This provides three Physical Simulators for _Silicon Dangling Bond (SiDB)_ Logic: [_ClusterComplete_](https://www.math.ru.nl/~bosma/Students/WillemLambooyMSc.pdf), [
+_QuickExact_](https://www.cda.cit.tum.de/files/eda/2024_aspdac_efficient_exact_simulation.pdf)
+and [_QuickSim_](https://www.cda.cit.tum.de/files/eda/2023_ieeenano_quicksim_physical_simulation.pdf).
 
 #### General information:
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)